operations/list.go

```diff
@@ -0,0 +1,301 @@
+package operations
```

Comment:
compile errors:

operations/list.go:73:23: too many arguments in call to listAliases
	have (context.Context, string, string, string, string)
	want (context.Context, string, string, string)
operations/list.go:220:2: undefined: comm
operations/list.go:228:20: undefined: ac
operations/list.go:229:18: undefined: v2
operations/list.go:229:38: undefined: lc
operations/list.go:238:18: undefined: comm
operations/list.go:250:1: missing return at end of function